src/vue/pages/assets/AssetDetails.vue

```diff
@@ -270,6 +272,11 @@ export default {
 
   margin-bottom: 2rem;
   width: 18rem;
+
+  &--disabled {
+    width: 21rem;
+    white-space: nowrap;
+  }
```

Comment:
bad, every time when the button become disabled it will be extended for no reason. i.e.: while the request being submitted. Please reconsider the solution.